### PR TITLE
doc: Release notes 17.2.7 with updated dashboard info

### PR DESCRIPTION
**Pull request by nizamial09**
## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by yuriw — APPROVED** [review]

File: doc/releases/quincy.rst [hunk 1]
```diff
@@ -69,6 +69,10 @@ Notable Changes
   recovery procedure, the recovered files under `lost+found` directory can now
   be deleted.
 
+* Dashboard: There is a new Dashboard page with improved layout. Active alerts
```
Comment (anthonyeleven) [inline doc/releases/quincy.rst:72]: an improved layout

File: doc/releases/quincy.rst [hunk 2]
```diff
@@ -69,6 +69,10 @@ Notable Changes
...
+  and some important charts are now displayed inside cards. This can be disabled and
+  older dashboard can be bought back by setting ``ceph dashboard feature disable dashboard``.
+
```
Comment (anthonyeleven) [inline doc/releases/quincy.rst:75]: This new dashboard can be disabled and the older layout brought back by setting

**Review by anthonyeleven — APPROVED** [review]
Rephrasing suggestions

File: doc/releases/quincy.rst (hunk 1 above)
Comment (nizamial09) [reply doc/releases/quincy.rst]: updated!

File: doc/releases/quincy.rst (hunk 2 above)
Comment (nizamial09) [reply doc/releases/quincy.rst:75]: updated!

**Pull request merged**